flexy-osisoft-connector/src/com/hms/flexyosisoftconnector/OSIsoftConfig.java

```diff
@@ -66,7 +66,11 @@ public OSIsoftConfig(String configFile) throws JSONException {
        //For each tagname in the config file create a tag and add it to 
        //the arraylist of tags
        for(int i = 0; i < tagNames.length(); i++) {
-          tags.add(new Tag(tagNames.getString(i)));
+          Tag tag = new Tag(tagNames.getString(i));
+          if (tag.isValidTag()) 

```

Comment:
should you report via a log message that the requested tag wasn't valid and was skipped?

Reply:
Invalid tags are logged in Tag.java at the point at which they are deemed invalid.

Reply:
It's always a questions of localization for logging. I wonder if that code should log the invalid tag at a low level like trace because who knows what the callers intent is whereas this code should log at info or error because at this level we know its an issue.

Reply:
Yeah ill change that

Reply:
Changed